src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -110,6 +110,26 @@
  *   &lt;property name=&quot;reliefPattern&quot; value=&quot;continue in next case&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Violation</p>
```

Comment:
Instead of "Violation" and "acceptable code", these can follow a pattern of showing a configuration, and then "Example:"